packages/contractkit/src/wrappers/Exchange.ts

```diff
@@ -123,4 +123,19 @@ export class ExchangeWrapper extends BaseWrapper<Exchange> {
    * @return The corresponding cUsd amount.
    */
   quoteGoldBuy = (buyAmount: NumberLike) => this.getSellTokenAmount(buyAmount, true)
+
+  /**
+   * Returns the exchange rate estimated at buyAmount.
+   * @param sellGold `true` if gold is the sell token
+   * @param buyAmount The amount of buyToken to estimate the exchange rate at (optional,
+   * defaults to large buyAmount to show worst case rate)
+   * @return The exchange rate (number of sellTokens received for one buyToken).
+   */
+  async getExchangeRate(
+    sellGold: boolean,
+    buyAmount: NumberLike = new BigNumber(1000 * 1000000000000000000) // in Wei
```

Comment:
does it make sense to have a default buyAmount?

Reply:
Yeah, generally the exchangeRate doesn't vary much at average amounts transacted in the app (around 10 dollars), so it's nice not to have to repeatedly pass in an amount that doesn't affect the rate much

Reply:
Actually, after thinking about this more, I think the caller should decide what their default rate is. Will remove it.